styles/dark/all-channels.mcss

```diff
@@ -60,6 +60,7 @@ AllChannels {
       flex: 1
       white-space: nowrap;
       min-width: 0;
+      -webkit-mask-image: linear-gradient(90deg, rgba(0,0,0,1) 90%, rgba(0,0,0,0));
```

Comment:
This is the source of the scroll perf hit. Seems to be okay on the Public sidebar, but when rendering thousands of them, things are getting pretty janky on my machine.

<img width="1123" alt="screen shot 2018-04-03 at 12 42 02 pm" src="https://user-images.githubusercontent.com/66834/38222998-787494d0-373c-11e8-9b61-7455146765fb.png">

I suggest replacing this code with a `text-overflow: ellipse` or something similar. For example:

```css
      text-overflow: ellipsis;
      overflow: hidden;
```